pkg/collector/corechecks/snmp/session.go

```diff
@@ -131,10 +131,13 @@ func fetchSysObjectID(session sessionAPI) (string, error) {
 		return "", fmt.Errorf("expected 1 value, but got %d: variables=%v", len(result.Variables), result.Variables)
 	}
 	pduVar := result.Variables[0]
-	_, value, err := getValueFromPDU(pduVar)
+	oid, value, err := getValueFromPDU(pduVar)
```

Comment:
does this mean that
1. you fetch OID `1.3.6.1.2.1.1.2.0` (sysObjectIDoid)
2. you get an OID that are different from the fetched OID
?

Reply:
Yes, that's right.

Devices might return for example `1.3.6.1.6.3.15.1.1.1.0` (usmStatsUnsupportedSecLevels) with a Counter value indicating the number of time this error happen (each call with return a incremented value)